packages/runner-docker/scripts/start.mjs

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env node
```

Comment:
This file does not need to be re-written. The container is always linux and thus everything going on in that container will be cross-system compatible. 

If you want to make the case that bash is harder to deal with than JS, then lets discuss that in a thread dedicated to the topic, but I would like to see a PR that is *JUST* the changes that add windows compatibility.